lib/mixins/fonts.js

```diff
@@ -88,5 +102,80 @@ export default {
     };
 
     return this;
+  },
+
+  /**
+   * Convert a {@link Size} into a point measurement
+   *
+   * @param {Size | boolean | undefined} size - The size to convert
+   * @param {Size | boolean | undefined} defaultValue - The default value when undefined
+   * @param {PDFPage} page - The page used for computing font sizes
+   *
+   * @returns number
+   */
+  sizeToPoint(size, defaultValue = 0, page = this.page) {
+    if (typeof defaultValue !== 'number')
+      defaultValue = this.sizeToPoint(defaultValue);
+    if (size === undefined) return defaultValue;
+    if (typeof size === 'number') {
+      if (size > 0) return size;
+      return defaultValue;
+    }
+    if (typeof size === 'boolean') return Number(size);
```

Comment:
What situation a boolean value makes sense?

Reply:
Its mainly for a falsy value of saying `margin: false` to indicate that you don't want a margin to render at all